nanoc-dart-sass/spec/nanoc/dart_sass/filter_spec.rb

```diff
@@ -57,6 +57,24 @@
     expect(res.strip).to match(/color: #333/m)
   end
 
+  it 'supports source_map' do
+    # Create item
+    ctx.create_item('stuff', {}, '/foo.sass')
+    ctx.create_rep(ctx.items['/foo.sass'], '/assets/foo.css')
+    ctx.item = ctx.items['/foo.sass']
+
+    filter = described_class.new(ctx.assigns)
+
+    res = filter.run(<<~SASS, source_map: true)
```

Comment:
(We may want to add a test to check that the source map does _not_ get included when the option is off)